presto-docs/src/main/sphinx/admin/tuning.rst

```diff
@@ -10,29 +10,392 @@ Config Properties
++                                          +                                          +                                    +worker then ``task.min-drivers`` OR the task has been  +
++                                          +                                          +                                    +enqueued with ``force start`` property.                +
++------------------------------------------+------------------------------------------+------------------------------------+-------------------------------------------------------+
++  ``task.operator-pre-allocated-memory``  +          ``String`` (data size)          +             ``16 MB``              +Memory preallocated for each driver in query           +
++                                          +                                          +                                    +execution. Increasing this value may cause less        +
++                                          +                                          +                                    +efficient memory usage but allows to fail fast in low  +
++                                          +                                          +                                    +memory environment more frequently.                    +
++------------------------------------------+------------------------------------------+------------------------------------+-------------------------------------------------------+
++       ``task.share-index-loading``       +               ``Boolean``                +             ``false``              +It allows to control whether index lookups join has    +
++                                          +                                          +                                    +index shared within a task. This enables the           +
++                                          +                                          +                                    +possibility of optimizing for index cache hits or for  +
++                                          +                                          +                                    +more CPU parallelism depending on the property value.  +
++                                          +                                          +                                    +Serves as default for ``task_share_index_loading``     +
++                                          +                                          +                                    +session property.                                      +
++------------------------------------------+------------------------------------------+------------------------------------+-------------------------------------------------------+
++          ``task.writer-count``           +               ``Integer``                +               ``1``                +Defines default number of writers per task. Serves as  +
```

Comment:
Does it really do that? I think at the end it's effectively defined by ``task_writer_count``.
Also, I think if it defines number of writers per task it does not cap the worker parallelism as it's virtually infinite being ``task_writer_count`` * number_of_tasks_per_worker.